src/BinaryKits.Zpl.Viewer/Helpers/FontScale.cs

```diff
@@ -0,0 +1,163 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace BinaryKits.Zpl.Viewer.Helpers
+{
+    // Offset is Intercharacter Gap (in dots), 0 if proportional
+    using FontScaleDictionary = Dictionary<string, (int height, int width, int offset)>;
+
+    internal static class FontScale
+    {
+        private static readonly FontScaleDictionary fontScales6mm = new()
+        {
+            ["A"] = (9, 5, 1),
+            ["B"] = (11, 7, 2),
+            ["C"] = (18, 10, 2),
+            ["D"] = (18, 10, 2),
+            ["E"] = (21, 10, 5),
+            ["F"] = (26, 13, 3),
+            ["G"] = (60, 40, 8),
+            ["H"] = (17, 11, 6),
+            ["GS"] = (24, 24, 0)
+        };
```

Comment:
Offset of E should be 3 (= 2.16 * 6 - 10), offset of H should be 4 (= 2.48 * 6 - 11). I didn't verify the other tables.

Reply:
No: they are in dots as you can see in [this table](https://docs.zebra.com/us/en/printers/software/zpl-pg/c-zpl-font-barcodes-fonts-andbar-codes/r-zpl-font-barcodes-proportional-fixed-spacing.html)

Reply:
Correct, in dots. 2.1667 mm * 6 dots/mm - 10 dots = 3 dots. The table appears to be specifically for 8 dpmm, though not labelled as such.